### PR TITLE
minor edits

### PR DESCRIPTION
**Pull request by geirman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ You should use stateless functions when your component
 - has no `refs`.
 - doesn't need a `constructor`.
 - doesn't access `context`.
+- doesn't need access to component lifecycle methods
```
Comment (timche) [inline README.md]: Could you please add a `.` at the end, so it matches the style?

**Comment by timche** [comment]
Thanks for your pull request! :) Just a small thing to add and I will merge this 👍🏻

**Comment by geirman** [comment]
Awesome, thanks for taking lead on this

**Comment by timche** [comment]
You are welcome! Happy to see people contributing :)

**Pull request merged**